ci-operator/config/openshift/sandboxed-containers-operator/openshift-sandboxed-containers-operator-devel__downstream-release.yaml

```diff
@@ -4,15 +4,15 @@ base_images:
     namespace: ci
     tag: "4.20"
   upi-installer:
-    name: "4.18"
+    name: "4.19"
```

Comment:
separate commit please

Reply:
also the same question as before, do you want to keep updating these? I think they can stay as they are until they are required to be bumped to allow running PR checks. But individual versions should be subject of the PRs that will cover the individual release testing so updating them regularly is unnecessary burden.